<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

        <node pkg="osrt_ros" type="OnlineLowerLimb" 	name="ik_lowerbody_node" output="screen" >
		<!-- param name="resample_period" 		value="0.01" / -->
		<param name="subject_dir" 		value="/gait1992/" />
		<param name="model_file" 		value="scale/model_scaled.osim" />
		<param name="imu_base_body" 		value="pelvis" />
		<rosparam param="imu_observation_order" >['torso','pelvis','femur_r','femur_l','tibia_r','tibia_l','talus_r','talus_l']</rosparam>
		<param name="imu_direction_axis" 		value="z" />
		<param name="imu_ground_rotation_x" 		value="-90" />
		<param name="imu_ground_rotation_y" 		value="-90" />
		<param name="imu_ground_rotation_z" 		value="0" />
		<param name="rate" 				value="400" />
		<param name="logger_filename_ik" 		value="_ik_lower" />
		<param name="logger_filename_imus" 		value="_imus_lower" />
		<param name="data_save_dir" 		value="/tmp/test" />
		
		<!-- remap from="/agrf_node/input" to="/inverse_kinematics_from_file/r_data"/>
		<remap from="/in_labels" to="/inverse_kinematics_from_file/out_labels"/ -->
        </node>

</launch>

